frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -78,9 +123,37 @@ onUnmounted(() => {
     resetNotification();
 });
 
+function sortByNewUserAndUserName(
+    first: UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew,
+    second: UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew
+) {
+    // First, sort by isNewUser (true first)
+    if (first.isNewUser && !second.isNewUser) return -1;
+    if (!first.isNewUser && second.isNewUser) return 1;
+    return 0;
+}
+
+const resetNewAddedTag = () => {
```

Comment:
so if we check if user is new or not in the table component, we don't need these anymore. So you just need to reset the newUserInTable to be ""